### ISSUE TITLE
Helper functions for logging `Show a => a` values

### ISSUE THREAD

**Issue by bitemyapp** [kazu-yamamoto/logger#77]
Thank you for the nice logging library! I use it a lot :)

![screenshot from 2016-02-01 20-04-19](https://cloud.githubusercontent.com/assets/320177/12737539/01a8cc50-c91f-11e5-8aac-5d40849712c4.png)

I end up writing this a lot. Would you be open to `$log*T` and `$log*TS` variants of the log functions which have the type:

```
$logDebugT :: (Show a, MonadLogger m) => a -> m ()
```

Instead of:

```haskell
$logDebug :: MonadLogger m => Text -> m ()
```

If this is something you're willing to do, let me know and I will PR it. I'm not stuck on the naming either.

**Comment by snoyberg** [comment]
I'm down with it. I'm not in love with the `T` suffix, but also don't have
any better ideas.

**Comment by bitemyapp** [comment]
@snoyberg would've used S if it hadn't been taken, for "Showable". Could use C for convert. Perhaps SH for SHow? Anyhoodle, I'll get the PR together and we can discuss there.

Thank you!

**Comment by snoyberg** [comment]
I like SH :)

**Comment by snoyberg** [comment]
Resolved by #78
